charts/spire/charts/spire-server/templates/tornjak-ingress.tpl

```diff
@@ -0,0 +1,54 @@
+{{- if eq (.Values.tornjak.enabled | toString) "true" }}
+
+{{- if .Values.tornjak }}
+{{- if .Values.tornjak.config }}
+
+{{- if .Values.tornjak.config.frontend }}
+{{- if .Values.tornjak.config.frontend.ingress }}
```

Comment:
Should probably stub out tornjak.config.frontend config in values, so you don't need to check for all the things here.